openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -1242,7 +1228,7 @@ def add_library_block_static_asset_file(usage_key, file_path, file_content, user
     return LibraryXBlockStaticFile(
         path=file_path,
         url=site_root_url + local_path,
-        size=content.size,
+        size=sys.getsizeof(file_content),
```

Comment:
Since `file_content` is a `bytes`, please use `len(file_content)` here. Using `sys.getsizeof` is going to get you the bytes used by the system internal representation of the variable, including things like pointers and object overhead.

Reply:
```python
>>> sys.getsizeof(b'')
33
```